### ISSUE TITLE
Incorrect UTC time on the clock returned from Alpaca.Markets.AlpacaTradingClient.GetClockAsync()

### ISSUE THREAD

**Issue by rudeGit** [alpacahq/alpaca-trade-api-csharp#235]
I am executing the following code (from Nuget library version 3.8.1) from mountain time zone on 9/20/2020 around 12:57PM (There is a 3 hour difference between mountain and easter time zones).
   
`var clock = Alpaca.Markets.AlpacaTradingClient.GetClockAsync();`

_clock.NextOpen 		=> Shows 9/21/2020 6:30:00 AM  (Deprecated)
clock.NextOpenUtc 	=> Shows 9/21/2020 6:30:00 AM_

The value in clock.NextOpen is **correct** (as it is local to mountain time zone) and the value in clock.NextOpenUtc **incorrect** as it should be universal time value that represents the DateTime representation of string value "2020-09-21T13:30:00.0000000Z"

Is it fixed in later versions? If so, which version should I use to get the correct value in NextOpenUtc property?

**Comment by OlegRa** [comment]
@rudeGit Could you please re-test this issue in the version [3.8.2](https://github.com/alpacahq/alpaca-trade-api-csharp/releases/tag/v3.8.2) available also on NuGet. If the problem is resolved - please, close this issue.

**Comment by rudeGit** [comment]
@OlegRa I tested with 3.8.2. The issue still remains as the problem is not fixed.

**Comment by OlegRa** [comment]
@rudeGit Please, test the updated version [3.8.3](https://www.nuget.org/packages/Alpaca.Markets/3.8.3) - it should handle all time formats, used by API properly (I hope).

**Comment by rudeGit** [comment]
@OlegRa It is still not fixed in 3.8.3 version also. I still see following:

```
clock .NextOpen.ToString()
"9/24/2020 1:30:00 PM"

clock .NextOpenUtc.ToString()
"9/24/2020 1:30:00 PM"
```

For comparison, see following:

```
DateTime.Now.ToString()
"9/23/2020 10:43:03 PM"

DateTime.UtcNow.ToString()
"9/24/2020 5:43:11 AM"
```

**Comment by OlegRa** [comment]
@rudeGit In fact, it works. Just check the result of the next code snippet:

```cs
var clock = await _alpacaTradingClient.GetClockAsync();

Console.WriteLine("DateTime.Now: {0:O}", DateTime.Now);
Console.WriteLine("DateTime.UtcNow: {0:O}", DateTime.UtcNow);

Console.WriteLine("clock.TimestampUtc: {0:O}", clock.TimestampUtc);
Console.WriteLine("clock.NextOpenUtc: {0:O}", clock.NextOpenUtc);
Console.WriteLine("clock.NextCloseUtc: {0:O}", clock.NextCloseUtc);

Console.WriteLine("clock.TimestampUtc.ToLocalTime(): {0:O}", clock.TimestampUtc.ToLocalTime());
Console.WriteLine("clock.NextOpenUtc.ToLocalTime(): {0:O}", clock.NextOpenUtc.ToLocalTime());
Console.WriteLine("clock.NextCloseUtc.ToLocalTime(): {0:O}", clock.NextCloseUtc.ToLocalTime());
```

on my box running in European Summer Time (UTC+2) I see the next results (I just add line feeds for readability):

```
DateTime.Now: 2020-09-24T10:25:44.6740789+02:00
DateTime.UtcNow: 2020-09-24T08:25:44.6757364Z

clock.TimestampUtc: 2020-09-24T08:25:45.7151747Z
clock.NextOpenUtc: 2020-09-24T13:30:00.0000000Z
clock.NextCloseUtc: 2020-09-24T20:00:00.0000000Z

clock.TimestampUtc.ToLocalTime(): 2020-09-24T10:25:45.7151747+02:00
clock.NextOpenUtc.ToLocalTime(): 2020-09-24T15:30:00.0000000+02:00
clock.NextCloseUtc.ToLocalTime(): 2020-09-24T22:00:00.0000000+02:00
```

As you can see the `TimestampUtc` value very close to my local time value (it's good). The `NextOpenUtc` and `NextCloseUtc` values also are also looking fine because trading session right now is in the **closed** state and the **next** trading day will start in 5 hours. Right after that time point this API call will return the new `NextOpenUtc` value but the same `NextCloseUtc` value because trading session will be in the **opened** state.

You shouldn't assume that `NextOpenUtc` is always less than `NextCloseUtc`. The only invariant that correct is the next:

```cs
Assert.True(clock.NextOpenUtc > clock.TimestampUtc);
Assert.True(clock.NextCloseUtc > clock.TimestampUtc);
```

**Comment by OlegRa** [comment]
@rudeGit The same code running right now (trading session is opened) generates the next results:

```
DateTime.Now: 2020-09-24T16:04:56.0077084+02:00
DateTime.UtcNow: 2020-09-24T14:04:56.0093591Z

clock.TimestampUtc: 2020-09-24T14:04:56.5070871Z
clock.NextOpenUtc: 2020-09-25T13:30:00.0000000Z
clock.NextCloseUtc: 2020-09-24T20:00:00.0000000Z

clock.TimestampUtc.ToLocalTime(): 2020-09-24T16:04:56.5070871+02:00
clock.NextOpenUtc.ToLocalTime(): 2020-09-25T15:30:00.0000000+02:00
clock.NextCloseUtc.ToLocalTime(): 2020-09-24T22:00:00.0000000+02:00
```

As you can see the `NextOpenUtc` returns tomorrow date/time now (tomorrow session open time) but the `NextCloseUtc` still returns today date/time (current session close time).

**Comment by rudeGit** [comment]
@OlegRa You are correct that problem was fixed in 3.8.3. My mistake was that I was hung up on the fact that both NextOpen and NextOpenUtc were returning the same value. What I failed to realize was that NextOpen property is irrelevant going forward (as it is marked obsolete) and I should only consider NextOpenUtc and that I need to call ToLocalTime method on it to get the local time.

Thank you for the detailed explanation.
